{{ cookiecutter.package_name }}/.readthedocs.yml

```diff
@@ -4,7 +4,7 @@ build:
   image: latest
 
 python:
-  version: 3.7
+  version: '{{ cookiecutter.minimum_python_version }}'
```

Comment:
This won't work. RTD has changed the way they specify Python version for 3.8 or later.

https://github.com/astropy/astropy/blob/main/.readthedocs.yml

Reply:
Good catch! Please take another look.